include/networks.h

```diff
@@ -12,7 +12,6 @@
 #include <math.h>
 #include <string.h>
 #include "datastructures.h"
-#include <pthread.h>
```

Comment:
Need this for parallelism?

Reply:
Yeah I'm just wondering if there are other modifications like this made, and I'm wondering if Dr. Boyles is up to date. We can prob verify this though.